deploy/charts/cert-manager/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: cert-manager
 # The version and appVersion fields are set automatically by the release tool
-version: v0.1.0
+version: v0.1.1
```

Comment:
Revert this please since the version will be auto-incresed when a release is made.